src/views/BannerSubmission/index.js

```diff
@@ -12,47 +48,484 @@ const style = {
-    <Grid container justifyContent="center">
-      <Grid item xs={12} lg={8}>
-        <Card>
-          <CardHeader
-            title="Advertise your club or event on the 360 Homepage!"
-            titleTypographyProps={{ variant: 'h4', align: 'center' }}
-            style={{
-              backgroundColor: gordonColors.primary.blue,
-              color: 'white',
-            }}
-          />
-          <CardContent>
-            <Grid container justifyContent="center" direction="column">
-              <Grid item align="left">
-                <Typography variant="h6">Banner Image Guidelines</Typography>
-                <Typography variant="body2">
-                  1. Attach JPG image with a resolution of 1500 by 600.
-                  <br />
-                  2. Text must be clearly legible.
-                  <br />
-                  3. Include a url that you would like the banner image to link to in your email.
-                  <br />
-                  4. All banner images must be approved. There is limited space, so not all images
-                  will be.
-                </Typography>
-              </Grid>
-              <Grid item align="center">
-                <a href="mailto:360@gordon.edu?Subject=Banner Image Submission">
-                  <Button variant="contained" style={style.uploadButton}>
-                    Email the 360 Team
-                  </Button>
-                </a>
-              </Grid>
+  const authenticated = useAuth();
+  const [currentUsername, setCurrentUsername] = useState('');
+  const isOnline = useNetworkStatus();
+  const [isAdmin, setIsAdmin] = useState(false);
+  const [loading, setLoading] = useState(true);
+
+  //Solely for banner management
+  const [banners, setBanners] = useState([]);
+  const [newBannerTitle, setNewBannerTitle] = useState('');
+  const [newBannerWebLink, setNewBannerWebLink] = useState('');
+  const [newBannerSortOrder, setNewBannerSortOrder] = useState(-1);
+  const [openBannerActivity, setOpenBannerActivity] = useState(false);
+
+  //Solely for photo functions
+  const [cropperImageData, setCropperImageData] = useState(null); //null if no picture chosen, else contains picture
+  const [photoDialogErrorTimeout, setPhotoDialogErrorTimeout] = useState(null);
+  const [photoDialogError, setPhotoDialogError] = useState(null);
+  const [snackbar, setSnackbar] = useState({ open: false, text: '', severity: '' });
+  const cropperRef = useRef();
+  const [aspectRatio, setAspectRatio] = useState(null);
+
+  const loadBanners = useCallback(async () => {
+    setLoading(true);
+    if (authenticated) {
+      const existingBanners = await cmsService.getSlides();
+      setLoading(false);
+      setBanners(existingBanners);
+    } else {
+    }
+  }, [authenticated]);
+
+  useEffect(() => {
+    loadBanners();
+  }, [authenticated, loadBanners]);
+
+  useEffect(() => {
+    const loadUsername = async () => {
+      const user = await userService.getProfileInfo();
```

Comment:
Yes, the preferred way to access user info is from the useUser hook, because it prevents re-fetching the same profile info from the API over and over.